README.md

```diff
@@ -22,7 +23,7 @@ Supported Systems
 Overview
 --------
 
-Chromebooks with Chrome OS run a Linux kernel. The only missing pieces to use them as full-featured Linux distro were gcc and make with their dependencies. Well, these pieces aren't missing anymore. Say hello to chromebrew!
+Chromebooks with Chrome OS run a Linux kernel. The only missing pieces to use them as full-featured Linux distro were gcc and make with their dependencies. Well, these pieces aren't missing anymore. Say hello to chromebrew! Gui Apps are Kinda working.
 
```

Comment:
How about `Most gui apps are working with sommelier.  We don't support i686, however.`?